be/src/olap/tablet_schema.cpp

```diff
@@ -337,14 +337,12 @@ void TabletColumn::to_schema_pb(ColumnPB* column) {
     }
 }
 
-TabletSchema::TabletSchema()
-    : _num_columns(0),
-      _num_key_columns(0),
-      _num_null_columns(0),
-      _num_short_key_columns(0) { }
-
 void TabletSchema::init_from_pb(const TabletSchemaPB& schema) {
     _keys_type = schema.keys_type();
+    _num_columns = 0;
+    _num_key_columns = 0;
+    _num_null_columns = 0;
```

Comment:
I am not sure, but did you miss the `_num_short_key_columns` here?

Reply:
@morningman `_num_short_key_columns` will be assigned in line 358 directly, not like the above variables which will be increased in the following loop, so I didn't reset it here.